docs/README.md

```diff
@@ -3,25 +3,24 @@
 ## What is the Enterprise Event Bus?
```

Comment:
We need a title on this page. That should look like: 
---
title: Enterprise Event Bus
---

Reply:
If you look at the mkdocs.yaml file, you can see how the Home page gets its title. If we use your example, the page will be named "Enterprise Event Bus" in the main nav. Marking this as resolved.